### PR TITLE
Add Gradle Usage to README

### LINKED ISSUE #16: Gradle usage example would be "nice to have".
Maven is by far leading Gradle in terms of usage ( https://gradle.org/maven-vs-gradle ), so it is understandable why Gradle usage is not addressed here (yet).

That said, Gradle is used w/i a significant number of projects, including some that I contribute to, so I have created such an example:
https://github.com/paegun/metrics-aspectj-gradle-example

Please feel free to link to the example or bring such an example under your control so it can more easily remain up-to-date w/ metrics-aspectj .

### PR DESCRIPTION
**Pull request by paegun**
* added a gradle usage section to README w/ a link to a working example.
  * resolves #16

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,6 +102,49 @@ Other options are detailed in the [AspectJ Ant tasks][] documentation.
...
 [AspectJ Ant tasks]: http://www.eclipse.org/aspectj/doc/next/devguide/antTasks.html
 
+#### Using Gradle
+
+[gradle example]: https://github.com/paegun/metrics-aspectj-gradle-example
+A working [gradle example][] is available, but each integration point is described here.
+
+##### build.gradle snippets
+
+```java
```
Comment (astefanutti) [inline README.md]: There may be a more appropriate langage or use script by default

File: README.md [hunk 2]
```diff
@@ -102,6 +102,49 @@ Other options are detailed in the [AspectJ Ant tasks][] documentation.
...
+// ensure Dropwizard Metrics as well as the aspect-oriented metrics (astefanutti.metrics.aspectj)
+//  runtime dependencies of your solution are satisfied.
+dependencies {
+    compile "io.astefanutti.metrics.aspectj:metrics-aspectj:${metricsAspectVer}"
+    // add a path for the gradle-aspectj "aspect weaving" (AspectJ Compiler compile)
+    aspectpath "io.astefanutti.metrics.aspectj:metrics-aspectj:${metricsAspectVer}"
+
+    compile "io.dropwizard.metrics:metrics-core:${metricsVer}"
+    compile "io.dropwizard.metrics:metrics-annotation:${metricsVer}"
+    compile "io.dropwizard.metrics:metrics-jersey2:${metricsVer}"
```
Comment (astefanutti) [inline README.md]: Is this dependency really required?

File: README.md (hunk 1 above)
Comment (paegun) [inline README.md]: A brief scour of gradle+markdown documents yielded `groovy` as the appropriate language.

File: README.md (hunk 2 above)
Comment (paegun) [inline README.md]: Oversight, removing.

**Comment by astefanutti** [comment]
I've just merge the PR. Thanks a lot!

**Pull request merged**